### PR TITLE
Configure metal core vlan

### PR DESCRIPTION
**Pull request by simcod**
Related to this PR: https://github.com/metal-stack/metal-core/pull/126

### CODE REVIEW

File: partition/partition-defaults/main.yaml [hunk 1]
```diff
@@ -16,3 +16,5 @@ metal_partition_metal_api_grpc_client_cert:
 metal_partition_metal_api_grpc_client_key:
 
 metal_partition_mgmt_gateway:
+
+metal_core_pxe_vlan: 966
```
Comment (simcod) [inline partition/partition-defaults/main.yaml:20]: Rename to `metal_core_pxe_vlan_id`

Comment (simcod) [reply partition/partition-defaults/main.yaml:20]: Need to be removed here and added into the mini-lab

**Comment by robertvolkmann** [comment]
`metal_core_pxe_vlan` is still undocumented.

**Comment by simcod** [comment]
Related to this PR: https://github.com/metal-stack/metal-core/pull/126

**Review by robertvolkmann — APPROVED** [review]

**Review by Gerrit91 — APPROVED** [review]

**Pull request merged**